tools/run_tests/artifact_targets.py

```diff
@@ -62,7 +62,7 @@ def create_jobspec(name, cmdline, environ=None, shell=False,
           cmdline=cmdline,
           environ=environ,
           shortname='build_artifact.%s' % (name),
-          timeout_seconds=10*60,
+          timeout_seconds=15*60,
```

Comment:
It's probably safe to write None here for artifact building: There's unlikely a case where this gets deadlocked (as there is when we're running unit tests), and so it'll prevent us having to tweak numbers going forward.